### PR TITLE
Fixes for CSV export and deleting old data upon restart

### PR DESCRIPTION
**Pull request by edwardstamps**

### CODE REVIEW

File: web/admin/src/Admin.js [hunk 1]
```diff
@@ -83,19 +82,26 @@ export default class Admin extends PureComponent {
...
     if (!this.state.sessionId) {
       const sessionIds = Object.keys(this.state.sessions)
-      if (sessionIds.length) this.setState({ sessionId: sessionIds[0] })
+      if (sessionIds.length) {
+        this.setState({ sessionId: sessionIds[0] })
+      }
+    }
+  }
+
+  checkSessionStatus = () => {
```
Comment (AdamLiechty) [inline web/admin/src/Admin.js]: To be more specific about what this function returns, maybe name it the more-specific `isSessionEnded` and reverse the true/false

File: web/admin/src/i18n/en-us/trivia.admin.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   "display": "Display Leaderboard",
   "end": "End Game",
   "reset": "Restart trivia session", 
-  "confirmReset": "Are you sure you want to restart the trivia session? This will reset all leaderboard data and cannot be undone.",
+  "confirmReset": "Are you sure you want to restart the trivia session? This will delete all leaderboard and user responses data and cannot be undone. Please export any relevant reports before hand.",
```
Comment (AdamLiechty) [inline web/admin/src/i18n/en-us/trivia.admin.json]: "beforehand" is one word
"user response data"

File: web/admin/src/Admin.js [hunk 3]
```diff
@@ -83,19 +82,26 @@ export default class Admin extends PureComponent {
...
+      }
+    }
+  }
+
+  isSessionEnded = () => {
+    const { publicSessions, sessionId } = this.state
+    if (publicSessions[sessionId]) {
+      const currentSession = publicSessions[sessionId]
+      if (currentSession.state === 'ENDED') {
+        return false
```
Comment (AdamLiechty) [inline web/admin/src/Admin.js]: Need to flip true/false for this function

Comment (edwardstamps) [reply web/admin/src/Admin.js]: That makes sense with the new function name but then I would need to do: const exportIsDisabled = !this.isSessionEnded()

Comment (AdamLiechty) [reply web/admin/src/Admin.js]: correct

**Pull request merged**